packages/admin-ui-codemod/src/button-review.js

```diff
@@ -0,0 +1,60 @@
+function replace(source, j, componentName) {
+  return j(source)
+    .find(j.JSXOpeningElement, { name: { name: componentName } })
+    .forEach((p) => {
+      const { name, attributes, selfClosing } = p.value
+
+      const newAttributes = attributes.map((attribute) => {
+        if (attribute.name.name === 'size') {
+          return j.jsxAttribute(
+            j.jsxIdentifier('size'),
+            j.literal(transformSize(attribute.value.value))
+          )
+        }
+
+        if (attribute.name.name === 'variant') {
+          return j.jsxAttribute(
+            j.jsxIdentifier('variant'),
+            j.literal(transformVariant(attribute.value.value))
+          )
+        }
+
+        return attribute
+      })
+
+      j(p).replaceWith(j.jsxOpeningElement(name, newAttributes, selfClosing))
+    })
+    .toSource()
+}
+
+function transformSize(value) {
+  if (!value) return 'normal'
+
+  return {
+    small: 'normal',
+    regular: 'normal',
+  }[value]
+}
+
+function transformVariant(value) {
+  if (!value) return 'primary'
+
+  return {
+    'adaptative-dark': 'neutralTertiary',
+    'adaptative-light': 'neutralTertiary',
```

Comment:
I'm not sure if someone is using this, but IMO we shouldn't add a replacement to this variant because it doesn't have an equivalent. Instead, we should let it break so the user can decide how to approach this